testsuite/features/build_validation/retail/proxy_container_branch_network.feature

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@proxy
+@private_net
+@containerized_server
+Feature: Prepare the containerized branch server for PXE booting
+  In order to use SUSE Manager for Retail solution in a containerized setup
+  As the system administrator
+  I prepare the branch network in containerized setup
+
+  Scenario: Activate the branch network on the proxy
+    When I activate the private network on the proxy
+
+  Scenario: The DHCP server works as expected
+    Then the "dhcp_dns" host should be present on private network
+    And name resolution should work on private network
```

Comment:
I would join these two:

```
  Scenario: Activate the branch network on the proxy
    When I activate the private network on the proxy
    Then the "dhcp_dns" host should be present on private network
    And name resolution should work on private network
```

Reply:
Hmmm... "I activate the branch network on the proxy" is probably not a good name.

It's more about enabling eth1 network interface than about the DHCP and DNS server. I realize this is misleading.

Maybe I need a better name, instead of joining.

Reply:
Done.

I also finally merged both scenarios after all.